xml/System.Text.RegularExpressions/CaptureCollection.xml

```diff
@@ -932,7 +1021,21 @@
       <Docs>
         <summary>Gets an object that can be used to synchronize access to the collection.</summary>
         <value>A object that can be used to synchronize access to the collection.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This member is an explicit interface member implementation. It can be used only when the <xref:System.Text.RegularExpressions.CaptureCollection> instance is cast to an <xref:System.Collections.ICollection> interface.
+ 
+ 
+ 
+ For collections whose underlying store is not publicly available, the expected implementation is to return the current instance. Note that the pointer to the current instance might not be sufficient for collections that wrap other collections; those should return the underlying collection's `SyncRoot` property.   
+ Most collection classes in the <xref:System.Collections?displayProperty=nameWithType> namespace also implement a `Synchronized` method, which provides a synchronized wrapper around the underlying collection. However, derived classes can provide their own synchronized version of the collection using the <xref:System.Collections.ICollection.SyncRoot%2A> property. The synchronizing code must perform operations on the <xref:System.Collections.ICollection.SyncRoot%2A> property of the collection, not directly on the collection. This ensures proper operation of collections that are derived from other objects. Specifically, it maintains proper synchronization with other threads that might be simultaneously modifying the collection instance.   
+ Enumerating through a collection is intrinsically not a thread-safe procedure. Even when a collection is synchronized, other threads can still modify the collection, which causes the enumerator to throw an exception. To guarantee thread safety during enumeration, you can either lock the collection during the entire enumeration or catch the exceptions resulting from changes made by other threads.   
+
```

Comment:
Can you make sure this explanation is specific for this API? Seems to be describing the interface method (and how it can be implemented by classes that inherit from the interface).